include/aikido/control/PositionCommandExecutor.hpp

```diff
@@ -0,0 +1,30 @@
+#ifndef AIKIDO_CONTROL_POSITIONCOMANDEXECUTOR_HPP_
+#define AIKIDO_CONTROL_POSITIONCOMANDEXECUTOR_HPP_
+#include <Eigen/Dense>
+#include <future>
+
+namespace aikido {
+namespace control {
+
+/// Abstract class for executing position commands.
```

Comment:
Nit: Elaborate that "a position command requests several DegreesOfFreedom to move to a specified configuration and provides feedback when the move is complete."